composer.json

```diff
@@ -25,6 +27,9 @@
         "ext-mbstring": "*"
     },
     "replace": {
+        "typo3/eel": "self.version",
```

Comment:
It is really required ?

Reply:
Just a result of the composer manifest merger script that probably wasn't executed before by someone... We can and should check these but that's not the responsibility of this change.